config/prow/cluster/starter.yaml

```diff
@@ -3,24 +3,149 @@
 # their own files.
 ---
 apiVersion: v1
+kind: Namespace
+metadata:
+  name: prow
+---
+apiVersion: v1
 kind: ConfigMap
 metadata:
-  namespace: default
+  namespace: prow
   name: plugins
 data:
-  plugins.yaml: ""
+  plugins.yaml: |
+    plugins:
+      << your_github_org >>:
+      - approve
+      - assign
+      - blunderbuss
+      - cat
+      - dog
+      - help
+      - heart
+      - hold
+      - label
+      - lgtm
+      - trigger
+      - verify-owners
+      - wip
+      - yuks
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: prow
+  name: github-token
+stringData:
+  token: <<insert-token-here>>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: prow
+  name: hmac-token
+stringData:
+  # Generate via `openssl rand -hex 20`. This is the secret used in the GitHub webhook configuration
+  hmac: << insert-hmac-token-here >>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: prow
+  name: s3-credentials
+stringData:
+  service-account.json: |
+    {
+      "region": "minio",
+      "access_key": "<<CHANGE_ME_MINIO_ACCESS_KEY>>",
```

Comment:
It doesn't seem like we should be encouraging people to include secrets in the starter.yaml file.

Reply:
Its not particularly nice but the thing is not publicly exposed, hence its not really a big deal IMHO

Reply:
Another option could be to use some kind of envsubst magic. But no idea if we want that/ if it's better.

Reply:
About secrets in starter.yaml:
- we have enterprise rules that we cannot commit secrets/passwords into git repos. 
- as a new Prow user, the new starter.yaml worked almost perfectly(Thank you!)  
- So following gitOps approach I wanted to check it in, so my team could start customizing, and so i could roll back when we mess up (remember we are new) But first i had to split out the secret creation.    It would have been better to have secret creation as a separate step in the first place.

Reply:
I disagree. The purpose of this is to get ASAP something that works so ppl can start toying around and check if Prow fits their requirements. If and how ppl solve the problem of not having this in their yaml is not something that is supposed to get solved here. Removing a secret from a yaml manifest is trivial. Adding it and with it the correct keys might not be as trivial.